crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -582,21 +578,4 @@ mod tests {
             }
         );
     }
-
-    #[test]
-    #[cfg(not(feature = "unstable-sharding"))]
-    fn test_config_default_network_config_no_sharding() {
-        let config = ConductorConfig::default();
-        assert_eq!(
-            config.network.tuning_params.arc_clamping(),
-            Some(kitsune_p2p::dht::arq::ArqClamping::Full)
```

Comment:
Should we not keep arc clamping? That was a minimal way to keep mobile development going while we replaced K1 and got sharding working

Reply:
Created: https://github.com/holochain/holochain/issues/4750